### PR TITLE
Add Transaction pending FAQ

### PR DESCRIPTION
**Pull request by CjS77**

### CODE REVIEW

File: _data/faq.yml [hunk 1]
```diff
@@ -55,4 +55,71 @@
...
+    1. Alice's wallet will then try and establish a connection to Bob directly.
+    1. If Bob responds, Alice sends over her half of the transaction.
+    1. Bob's wallet will check that the transaction looks legit, and then fill in his part of the transaction, sign
+    it, and then try and send it back to Alice. At this point, Bob's wallet will say "Waiting for sender to complete
+    transaction".
+    1. If Alice is still online and receives Bob's reply, she'll check that the transaction still checks out, signs
+    it and then attempts to broadcast the transaction to the blockchain network.
+    1. Alice's wallet sends a courtesy message back to Bob to say that she's done this. Now both wallets update the
+    transaction status to say  "Completing final processing".
+    1. One the transaction has been mined and is included in the blockchain, it can be marked as complete. Both
```
Comment (stringhandler) [inline _data/faq.yml]:
```suggestion
    1. Once the transaction has been mined and is included in the blockchain, it can be marked as complete. Both
```

Comment (philipr-za) [inline _data/faq.yml]:
```suggestion
    1. Once the transaction has been mined and is included in the blockchain, it can be marked as complete. Both
```

**Review by stringhandler — APPROVED** [review]

File: _data/faq.yml [hunk 2]
```diff
@@ -55,4 +55,71 @@
...
+    wallets need to periodically ask a base node whether the transaction has been mined yet, in order to know
+    whether that last step has happened.
+
+    When this goes smoothly, the transaction negotiation completes in under a second, and then it takes roughly two
+    minutes to get confirmed. But there are at least a dozen points in that flow where things can be disrupted.
+
+    * If either Alice or Bob go offline at any point in the above flow, or
+    * if either run iOS and put the app in the background, or
+    * if there's a problem with the ToR client making a connection to the network, or
```
Comment (philipr-za) [inline _data/faq.yml]:
```suggestion
    * if there's a problem with the Tor client making a connection to the network, or
```

File: _data/faq.yml (hunk 1 above)
Comment (CjS77) [reply _data/faq.yml]: Thanks, already caught that one

**Pull request merged**